docs/tools/vdb_table/data/aerospike.json

```diff
@@ -0,0 +1,191 @@
+{
+  "name": "Aerospike",
+  "links": {
+    "docs": "https://aerospike.com/docs/vector/",
+    "github": "https://github.com/aerospike/proximus-examples",
+    "website": "https://aerospike.com/vector/",
+    "vendor_discussion": "",
+    "poc_github": "https://github.com/hev",
+    "slug": "aerospike"
+  },
+  "oss": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "license": {
+    "value": "Proprietary",
+    "source_url": "",
+    "comment": ""
+  },
+  "dev_languages": {
+    "value": [
+      "java"
+    ],
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_launch_year": 2024,
+  "metadata_filter": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "facets": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://aerospike.com/docs/vector/architecture/vector-data",
```

Comment:
Per the linked [docs page](https://aerospike.com/docs/vector/architecture/vector-data):
> Aerospike supports up to 4096 sets, which can be allocated in any way across namespaces. Aerospike sets do not require updating an Aerospike node, so this makes them an ideal tool for separation of datasets. Additionally an index can include multiple sets, so you can combine this data for administrative or other scenarios where aggregation is useful.

I'm not sure that this means the same thing as "support for multiple indexes per document". This column in the table is intended for when the user can create/update multiple vectors for a given document and combine them in various ways (e.g., reranking) to improve the relevance of the result.

See the docs page for Milvus that describes this feature:
https://milvus.io/docs/multi-vector-search.md

Could you clarify this further, and if it doesn't meet the requirements, mark this field as "none"?

Reply:
Updated documentation and linked to the more specific section.